src/Exception/ApiProblemException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpro\ApiProblem\Exception;
+
+use Phpro\ApiProblem\ApiProblemInterface;
+
+class ApiProblemException extends \Exception
+{
+    /**
+     * @var ApiProblemInterface
+     */
+    private $apiProblem;
+
+    public function __construct(ApiProblemInterface $apiProblem)
+    {
+        $data = $apiProblem->toArray();
+        $message = $data['detail'] ?? ($data['title'] ?? '');
```

Comment:
Additional brackets are not needed here. `$d = $a ?? $b ?? $c;` works the same

Reply:
I agree. But I think it's more readable with the parentheses, but that's my opinion.